packages/flutter/lib/src/widgets/spacer.dart

```diff
@@ -2,18 +2,23 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import '../../rendering.dart';
 import 'basic.dart';
 import 'framework.dart';
 
 /// Spacer creates an adjustable, empty spacer that can be used to tune the
 /// spacing between widgets in a [Flex] container, like [Row] or [Column].
 ///
-/// The [Spacer] widget will take up any available space, so setting the
-/// [Flex.mainAxisAlignment] on a flex container that contains a [Spacer] to
-/// [MainAxisAlignment.spaceAround], [MainAxisAlignment.spaceBetween], or
-/// [MainAxisAlignment.spaceEvenly] will not have any visible effect: the
-/// [Spacer] has taken up all of the additional space, therefore there is none
-/// left to redistribute.
+/// The [Spacer] widget will take space in a dynamic or fixed way, depending
+/// on which constructor is used:
+/// * The default unnamed constructor [Spacer] will take up any available
+/// space, so setting the [Flex.mainAxisAlignment] on a flex container that
+/// contains a [Spacer] to [MainAxisAlignment.spaceAround],
+/// [MainAxisAlignment.spaceBetween], or [MainAxisAlignment.spaceEvenly] will
+/// not have any visible effect: the [Spacer] has taken up all of the
+/// additional space, therefore there is none left to redistribute.
+/// * The [Spacer.fixed] constructor will take an absolute amount of space in
+/// the main axis.
```

Comment:
nit: clean up the formatting of this enumeration (see other enumerations in the code base)

Reply:
Done.